src/Services/movie-service.js

```diff
@@ -1,70 +1,69 @@
 import axios from 'axios';
 
-const movieServiceUrl = "http://localhost:4000/";
+const movieServiceUrl = 'http://localhost:4000/';
 
 async function sendRequest(resourceUrl, method, requestBody = null) {
```

Comment:
Maybe it would be better to use function expressions instead of function declarations (point 7.1 https://github.com/airbnb/javascript#functions--declarations)?